src/pudl/extract/ferc1.py

```diff
@@ -213,429 +208,272 @@
 }
 """A mapping of PUDL DB table names to their XBRL and DBF source table names."""
```

Comment:
`TABLE_NAME_MAP` seems like another thing that we might want to relegate to `package_data`

Reply:
Yep. This might require some rework of both dbf and xbrl extractors.

Reply:
Okay, not a high priority for the moment I think. Just something to add to the cleanup list.

Reply:
I wonder if this rework could be done independently of this initial effort. WDYT?